### PR TITLE
Moved to new MPP format, kotlin dsl

### PR DESCRIPTION
**Pull request by leodamsky**
This PR is for issue #64 

As there is a discussion about Kotlin DSL in this issue, I've added it too(upgrading Gradle to 5.5.1 and Kotlin to 1.3.41)

Compatibility version of Kotlin in `kotlin-logging-jvm` I kept 1.1. Tried not to change an existing codebase to make easier to review the PR. It seems I fixed all backward-incompatible stuff, but review though, please.

### CODE REVIEW

**Comment by oshai** [comment]
Thanks! I will review it as soon as I have some time.

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import java.util.Date
+
+plugins {
+    kotlin("multiplatform") version "1.3.41"
+    id("com.jfrog.bintray") version "1.8.4"
+    `maven-publish`
+    `java-library` //todo: ask whether we need javadoc at all as it is always empty
```
Comment (leodamsky) [inline build.gradle.kts]: Do you need javadocs? The jars are empty

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        mavenPublication {
+            // make a name of jvm artifact backward-compatible, default "-jvm"
+            artifactId = rootProject.name
+        }
+    }
+    js {
+        compilations.named("main") {
+            kotlinOptions {
+                metaInfo = true
+                sourceMap = true
```
[the remark this answers is not in the document]
Comment (oshai) [reply build.gradle.kts]: ok

File: build.gradle.kts (hunk 1 above)
Comment (oshai) [reply build.gradle.kts]: If you check this you'll see jars are not empty: https://repo1.maven.org/maven2/io/github/microutils/kotlin-logging/1.6.26/

It is required by maven central

Comment (leodamsky) [reply build.gradle.kts]: I think [dokka](https://github.com/Kotlin/dokka) would help

Comment (leodamsky) [reply build.gradle.kts]: Sorry, by empty I meant jar contains only MANIFEST.MF

Comment (leodamsky) [reply build.gradle.kts]: But there are no javadocs themselves

**Comment by oshai** [comment]
Thanks, it looks good!
Before merging it, I will publish a release from the PR to make sure it works.
I guess we will have a separate issue for the dokka docs. I will open that later.

**Comment by oshai** [comment]
I released version 1.7.0-mpp, will test it and then merge. you're welcome to check also.

**Comment by oshai** [comment]
It looks good, I merged it. Thanks again!
Will release version 1.7.1 with it.

**Pull request merged**